src/javascript/common/champion.js

```diff
@@ -116,7 +116,7 @@ const Champion = (function() {
             'metatrader-5'         : { module: MT5 },
             'champion-trader'      : { module: ClientType },
             'economic-calendar'    : { module: ClientType },
-            'mt5-types-of-accounts': {},
+            'mt5-types-of-accounts': { module: MT5 },
```

Comment:
i think it is safe to remove this line since it is now a static page?

Reply:
@kellybinary but it is calling .tabs on its onload function (https://github.com/ChampionFX/champion-static/pull/268/files#diff-f6da2a8419af33e18bf32125df3b4c1aL9) so we need it, right?